locales/en_US.json

```diff
@@ -62,12 +62,12 @@
 				"answer": "No. Lots of Pretendo's features/services are developed independently (for example, Miiverse may be worked on by one developer while Accounts and Friends is being worked on by another) and therefore we cannot give an overall ETA for how long this will take."
 			},
 			{
-				"question": "Does Pretendo work on Cemu/emulators?",
-				"answer": "Pretendo supports any client that can interact with Nintendo Network. Currently the only emulator with this kind of functionality is Cemu. Cemu 2.0 officially supports Pretendo under your network account options in the emulator. For information on how to get started with Cemu, check out the <a href='https://pretendo.network/docs/install/cemu'>documentation</a>.<br>Citra does not support true online play and thus does not work with Pretendo, and does not show signs of supporting true online play at all. Mikage, a 3DS emulator for mobile devices, may provide support in the future though this is far from certain."
+				"question": "When will you add more games?",
+				"answer": "We work on new games once we feel that our backend libraries are ready to support it, and there is developer interest in maintaining it. A lot of our work goes into stabilising and completing our existing games - we want to get the best experience possible in those before we move on to new titles. Since new work comes up all the time, we cannot make any estimate of when that would be."
 			},
 			{
-				"question": "If I am banned on Nintendo Network, will I stay banned when using Pretendo?",
-				"answer": "We will not have access to Nintendo Network's bans, and all users will not be banned on our service. However, we will have rules to follow when using the service and failing to follow these rules could result in a ban."
+				"question": "Does Pretendo work on Cemu/emulators?",
+				"answer": "Cemu 2.0 officially supports Pretendo under your network account options in the emulator. For information on how to get started with Cemu, check out the <a href='https://pretendo.network/docs/install/cemu'>documentation</a>.<br>Citra does not support true online play and thus does not work with Pretendo, and does not show signs of supporting true online play at all. Mikage, a 3DS emulator for mobile devices, may provide support in the future though this is far from certain."
```

Comment:
Since this PR was made Cemu has released version 2.1. This answer was always a bit misleading since "Cemu 2.0" actually does NOT support us (Cemu 2.0 is the stable release, and we got support in Cemu 2.0-5 which is an experimental release). However Cemu has no released 2.1, which is the first stable release to support us

Also I wonder if we should give a nebulous "some 3DS emulators/forks may support us" type of deal. Pablo's fork does work on our servers, but there's concerns from his end about Nintendo getting trigger happy (which is understandable, and why he doesn't provide any setup instructions himself). Also Citra is dead so we probably shouldn't talk about it like it's still alive